blockchains/ethereum/assets/0x84d4AfE150dA7Ea1165B9e45Ff8Ee4798d7C38DA/info.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DMM: USDT",
+    "website": "https://defimoneymarket.com.com",
```

Comment:
Please change `https://defimoneymarket.com.com` to `https://defimoneymarket.com`